README.md

```diff
@@ -1,43 +1,419 @@
-cloud-sdk-docker
-================
 
-> :warning: **IMPORTANT NOTE:** The base image for the `cloud-sdk` image will be changed to Alpine Linux on or after 5/15/17.
-> For more information, please see: [#58](https://github.com/GoogleCloudPlatform/cloud-sdk-docker/issues/58)
+# Google Cloud SDK Docker
+
+
+The [Cloud SDK](https://cloud.google.com/sdk/) is the command-line interface to the Google Cloud Platform.  It is a flexible utility which uses GCPs own Cloud APIs to perform many different tasks such as deploying code on AppEngine, to creating Compute Engine VMs to checking IAM permissions and so on.  It is the command line interface to pretty much every Google Cloud Platform API and service.   
+
+This article describes some more ways customers can use it by demonstrating several use cases involving containerizing the Cloud SDK.   The following shows Dockerfiles targeted towards various operating systems and some sample usecases (Its nothing new, just a sample set to document the baseline dockerfiles.
+
+* [Dockerfile](#dockerfile)
+    - [Ubuntu](#ubuntu)
```

Comment:
it's ok to have 2 flavors. so we can have 2 directories: alpine, debian. then we can build/document which tag corresponds to which image just like official images explain in their Docker Hub pages. no need to move this to the root of the repo.